applications/asset_tracker_v2/src/ext_sensors/ext_sensors.c

```diff
@@ -261,3 +266,26 @@ int ext_sensors_mov_thres_set(double threshold_new)
 
 	return 0;
 }
+
+int ext_sensors_accelerometer_trigger_callback_set(bool enable)
+{
+	int err;
+
```

Comment:
Blank lines between variable declarations and definitions are generally not needed